packages/protocol-kit/src/contracts/SafeWebAuthnSignerFactory/v1.4.1/SafeWebAuthnSignerFactoryContract_v1_4_1.ts

```diff
@@ -0,0 +1,73 @@
+import SafeWebAuthnSignerFactoryBaseContract from '@safe-global/protocol-kit/contracts/SafeWebAuthnSignerFactory/SafeWebAuthnSignerFactoryBaseContract'
+import {
+  SafeVersion,
+  SafeWebAuthnSignerFactoryContract_v1_4_1_Abi,
+  SafeWebAuthnSignerFactoryContract_v1_4_1_Contract,
+  SafeWebAuthnSignerFactoryContract_v1_4_1_Function,
+  SafeWebAuthnSignerFactory_1_4_1_ContractArtifacts
+} from '@safe-global/safe-core-sdk-types'
+import SafeProvider from '@safe-global/protocol-kit/SafeProvider'
+
+/**
+ * SafeWebAuthnSignerFactoryContract_v1_4_1  is the implementation specific to the Safe Proxy Factory contract version 1.4.1.
```

Comment:
The `Safe Proxy Factory` is mentioned in multiple comments here, let's change it to the correct name to avoid confusion

Reply:
done!